### DISCUSSION TITLE
Item Preview Panel for Transmogrification Menu

### DISCUSSION THREAD

**Discussion by armchairrockstar1** [chromiecraft/chromiecraft#6077] [Suggestions]
People are trying to use 3rd party addons to keep track of their transmoggable items, but it's easy for the devs to add an extra panel to Transmog Menu where all items are CTRL-clickable. On this secondary menu, the player can CTRL-click and preview his items in dressing room before deciding on mogging it.

Should be pretty easy to implement and it would be a huge quality of life improvement.

Thank you.

**Comment by 55Honey** [comment]
Duplicate of https://github.com/chromiecraft/chromiecraft/discussions/5768

**Comment by armchairrockstar1** [comment]
The addon Mogit worked well for a while until I accidentally deleted its lua files in WTF folder. Now filtering doesn't work because the addon thinks I don't own any items.

**Comment by armchairrockstar1** [comment]
Writing this here since someone interested can read or reply. I tried .transmog sync command today and it said it was successful. However I'm not sure what it did. I thought the server would tell the addon about all the items I own, and the addon would then show them to me when I use the filtering function. This didn't work as I expected.

Edit: Maybe this command is only working for TransmogTip rather than Mogit.

Edit 2: I fixed the Mogit problem by downgrading it to 1.5 from 1.7. Now it shows all items I've ever collected. Though a serverside interface or panel would still be preferable.
